### PR TITLE
[KOGITO-138] - Add notes for go v1.12.7 installation issue

### PR DESCRIPTION
**Pull request by xiezhang7**
Add notes for go installation issue via dnf on fedora.

https://issues.jboss.org/browse/KOGITO-138

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by xiezhang7** [comment]
@ricardozanini Please take a look. Thanks.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Kogito Operator was designed to deploy [Kogito Runtimes](https://github.com/kieg
 
 ## Requirements
 
-- go v1.12+
+- go v1.12+ (**Note:** The v1.12.7 go tools installed via [dnf](https://developer.fedoraproject.org/tech/languages/go/go-installation.html) on Fedora don't work with operator-sdk v0.9.0. Install with the [archive](https://golang.org/doc/install#install) instead.)
```
Comment (ricardozanini) [inline README.md]: @xiezhang7 do we have any evidence (besides our tests) of this problem? Maybe a github issue, a mail thread, or anything else in the community about this issue? Would be nice to add a link here to such discussion.

**Comment by ricardozanini** [comment]
Waiting for the fix in the repositories. Case this error spreads over the community, we add a link to the thread and merge this.

**Comment by ricardozanini** [comment]
Will be resolved as part of [KOGITO-857](https://issues.redhat.com/browse/KOGITO-857)

**Pull request closed without merging**